src/modules/sync/components/sync-dialog.tsx

```diff
@@ -218,13 +237,20 @@ export function SyncDialog( {
 		onRequestClose();
 	};
 
+	// Calculate dynamic padding based on number of notices
+	const noticeCount = [ isPushSelectionOverLimit, shouldShowVersionMismatch ].filter(
+		Boolean
+	).length;
+	const footerPadding =
+		noticeCount === 0 ? 'pb-[70px]' : noticeCount === 1 ? 'pb-[140px]' : 'pb-[210px]';
+
 	return (
 		<Modal
 			className="w-3/5 min-w-[550px] max-h-[84vh] [&>div]:!p-0"
 			onRequestClose={ onRequestClose }
 			title={ syncTexts.title }
 		>
-			<div className={ isPushSelectionOverLimit ? 'pb-[140px]' : 'pb-[70px]' }>
+			<div className={ footerPadding }>
```

Comment:
Not a strong opinion, but we could declare the footer padding inside the class. Something like:
```
			<div
				className={ cx(
					noticeCount === 0 && 'pb-[70px]',
					noticeCount === 1 && 'pb-[140px]',
					noticeCount >= 2 && 'pb-[210px]'
				) }
			>
```